code/modules/clothing/clothing.dm

```diff
@@ -2,6 +2,7 @@
 	name = "clothing"
 	var/eye_protection = 0 //used for headgear, masks, and glasses, to see how much they protect eyes from bright lights.
 	var/tint = TINT_NONE // headgear, mask and glasses, forvision impairment overlays
+	var/accuracy_mod = 0 //headgear that provides accuracy enhancing properties. -1 to 1 recommended
```

Comment:
Use discrete bonuses instead of multiplier. There are too many sources of accuracy modification so we should move away from multipliers to avoid spikiness.

Reply:
It isn't a multiplier. it is a flat bonus.

Reply:
Then maybe edit the comment? The recommendation of between -1 and +1 is misleading.

Reply:
Is it? it's a flat bonus to something that is a multiplier.

Reply:
```
gun_accuracy_mod += living_user.ranged_accuracy_mod
projectile_to_fire.accuracy = round((projectile_to_fire.accuracy * gun_accuracy_mult) + gun_accuracy_mod)
```
It's not, it's a flat bonus, and that's how we want to keep it.

Reply:
eh, gonna let future coders figure it out, no comments for them.